language/move-lang/src/parser/syntax.rs

```diff
@@ -619,15 +619,8 @@ fn parse_pack_or_call<'input>(tokens: &mut Lexer<'input>) -> Result<Exp_, ParseE
 fn parse_call_args<'input>(tokens: &mut Lexer<'input>) -> Result<Spanned<Vec<Exp>>, ParseError> {
     let start_loc = tokens.start_loc();
     consume_token(tokens, Tok::LParen)?;
-    let mut args = vec![];
-    while tokens.peek() != Tok::RParen {
-        args.push(parse_exp(tokens)?);
-        if tokens.peek() == Tok::RParen {
-            break;
-        }
-        consume_token(tokens, Tok::Comma)?;
-    }
-    tokens.advance()?; // consume the RParen
+    let args = parse_comma_list(tokens, Tok::RParen, parse_exp, true)?;
```

Comment:
I kind of don't like the trailing comma on ExpList, and I feel like I shouldn't like this one for consistency, but I'm unsure